spec/controllers/api/items_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe Api::ListsController do
+  describe 'POST create' do
+    let!(:user_1) { FactoryGirl.create(:user) }
+    let!(:list_1) { FactoryGirl.create(:list) }
+    before do
+      @request.env['HTTP_AUTHORIZATION'] = ActionController::HttpAuthentication::Basic.encode_credentials(user_1.username, user_1.password)
+      @new_item = build(:item)
+      post :create, item: { name: @new_item.name }
+    end
+    it 'returns http success' do
```

Comment:
Oh cool!  If you're paying attention to rubocop no need for both.

To setup HoundCI to behave like the rubocop default, create a file `.hound.yml` in your project root which looks like

```
ruby:
  config_file: .ruby-style.yml
```

the copy the default rubocop yml file I linked to into your repository as `.ruby-style.yml`